src/structures/GuildMember.js

```diff
@@ -230,6 +230,17 @@ class GuildMember extends Base {
   get displayName() {
     return this.nickname || this.user.username;
   }
+  
+  /**
+   * The display tag of a user, a normal tag if they don't have a nickname
+   * If someone's nickname is hydrabolt, but their username is hydar, displayTag will be hydrabolt#0001
```

Comment:
Typo here: displayTag will be hydar#0001
And I think it's more correct to use "somebody" instead of "someone".